### PR TITLE
Correct case render, edited out unneeded text

### PR DESCRIPTION
**Pull request by ZRock35**

### CODE REVIEW

File: diy/cases.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ The SlimeVR community has built a huge number of cases for style, internals and
 ## Arcturus by Lixulia
 Design by Lixulia
 
-![Case](https://raw.githubusercontent.com/Lixulia/Arcturus/main/renders/renderforarcturus.PNG) {: .caseImage}
+![Case](../assets/img/renderforarcturusnew.png)
```
Comment (Louka3000) [inline diy/cases.md:131]:
```suggestion
![Case](../assets/img/renderforarcturusnew.png) {: .caseImage}
```

**Review by Louka3000 — CHANGES_REQUESTED** [review]
`.caseImage` isn’t useless. It limits the image’s width to 400px

File: diy/cases.md (hunk 1 above)
Comment (ZRock35) [reply diy/cases.md:131]: that doesn't seem to work on my side! it just adds the text of .caseImage below

**Review by Louka3000 — APPROVED** [review]

File: diy/cases.md (hunk 1 above)
Comment (Louka3000) [reply diy/cases.md]: Because vscode doesn’t use it, but it works with Jekyll

**Review by calliePepper — APPROVED** [review]

**Pull request merged**